src/comments/handler.js

```diff
@@ -5,7 +5,7 @@ const {
   isBlockComment
 } = require('../prettier-comments/language-js/comments');
 
-const handleContractDefinitionComments = require('./handlers/ContractDefinition');
+const handlers = require('./handlers');
```

Comment:
can we `const { ContractDefinition, ModifierInvocation } = require('./handlers');` perhaps?

Reply:
We could. I just wanted to be very specific that these are handlers. I can change it if it's something we want.

Reply:
The function we invoke against these components seems self-explanatory to me "handleOwnLineComment" but I'm fine either way

Reply:
I just made a small change on this so we won't need to name every handler when evaluating.